pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = [
+    "hatchling",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "empack"
+description = "empack emscripten+boa"
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [
+    { name = "Thorsten Beier", email = "derthorstenbeier@gmail.com" },
+]
+keywords = [
+    "empack",
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "appdirs",
+    "networkx",
+    "pydantic",
+    "pydantic",
```

Comment:
pydantic is there twice, it was already the case before but since we're touching this file let's fix it

```suggestion
```